mysql-test/suite/perfschema/t/socket_instances_func.test

```diff
@@ -309,9 +309,9 @@ WHERE THREAD_ID = @thread_id;
 
 --echo #Compare server listener socket thread ids
 --disable_query_log ONCE
-eval SELECT ($server_tcpip_thread_id = $server_unix_thread_id) into @match_thread_id;
+#eval SELECT ($server_tcpip_thread_id = $server_unix_thread_id) into @match_thread_id;
```

Comment:
all these files will come back to normal, when i flip the default back. maybe I should create a diff with all this clutter removed.